### ISSUE TITLE
[Bug]: Static Page Generator Maintenance Task not working correctly when symfony framework ssi is enabled

### ISSUE THREAD

**Issue by melissakittl** [pimcore/pimcore#15983]
### Pimcore version

11.0.9

### Problem
Usage of the static page generation in combination with symfony server side includes functionality (https://symfony.com/doc/current/http_cache/ssi.html) on NGINX server does not work as expected in Maintenance StaticPageGenerationTask. 

### How to reproduce

1. enable ssi via symfony config.yaml and add nginx config to use ssi
```
framework:
    ssi: { enabled: true }
    fragments: { path: /_fragment_ssi }
```
2. create a template (`test.html.twig`), that calls: 
```{{ render_ssi( controller("App\\Controller\\TestController::testSSIAction") ) }}```

4. create a document in Pimcore with the template `test.html.twig` and enable server side static rendering
![image](https://github.com/pimcore/pimcore/assets/23357021/5a2bfcd2-f0e5-41a5-8d91-c6c9b51244ba)

5. run `bin/console pimcore:maintenance --job=documents_static_page_generate` and have a look at the generated static file (located under `/public/var/tmp/pages/{path-to-document}`)

### Actual Behavior

6. if the static page is generated with the [StaticPagesGenerationTask](https://github.com/pimcore/pimcore/blob/11.x/lib/Maintenance/Tasks/StaticPagesGenerationTask.php#L70), the final html is generated and saved in the static page:
![image](https://github.com/pimcore/pimcore/assets/23357021/ae147082-e21d-47ca-8c08-f9282dbbf271)

The DocumentRenderer, which gets used inside the StaticPageGenerator, does not consider the SSI functionality from symfony and always uses the InlineFragmentRenderer, see [configuration](https://github.com/pimcore/pimcore/blob/11.x/bundles/CoreBundle/config/documents.yaml#L52)

### Expected Behavior

5. The generated static page should contain a string like the following, where the `render_ssi` gets called: 
``
``

6. if the static page is generated via [StaticPageGeneratorListener](https://github.com/pimcore/pimcore/blob/11.x/bundles/CoreBundle/src/EventListener/Frontend/StaticPageGeneratorListener.php#L147)
it works as expected, and the includes for the ssi gets rendered instead of the final html.
![image](https://github.com/pimcore/pimcore/assets/23357021/ce49a4fc-5a81-424e-8e92-e51876f97ab8)

**Comment by jdreesen** [comment]
I assume that nginx sends some kind of header that lets Symfony know that the web server understands SSI. This header is obviously missing when the pages are generated via a maintenance task, because they are started from the console (like a cronjob). This makes Symfony think that the feature is not available, and it is disabled.

**Comment by NiklasBr** [comment]
This is similar to, but not a duplicate of, #14468.

**Comment by Gerhard-Kanzler** [comment]
Would fixed by: https://github.com/pimcore/pimcore/pull/16012

**Comment by dvesh3** [comment]
Fixed by #16012
